pkg/lightning/backend/local/local.go

```diff
@@ -101,7 +105,8 @@ const (
 	// the lower threshold of max open files for pebble db.
 	openFilesLowerThreshold = 128
 
-	duplicateDBName = "duplicates"
+	duplicateDBName       = "duplicates"
+	remoteDuplicateDBName = "remote_duplicates"
```

Comment:
`remoteDuplicateDBName` and `"remote_duplicates"` seems not used anywhere according to `git grep`